cli/command/formatter/context.go

```diff
@@ -73,6 +75,10 @@ func (c *clientContextContext) Name() string {
 	return c.c.Name
 }
 
+func (c *clientContextContext) Type() string {
+	return c.c.Type
```

Comment:
Thinking what would make more sense; set the default, or have this function return the default if it's empty 🤔